packages/server/src/models/objection/UserNotification.js

```diff
@@ -1,4 +1,6 @@
 import BaseModel from './BaseModel'
+const MINUTES = 60 * 1000
+const HOURS = 60 * MINUTES
```

Comment:
We can probably move this to a times helper module, I think we have declared these in multiple places already